src/pages/ReimbursementAccount/CompanyStep.js

```diff
@@ -303,7 +303,7 @@ class CompanyStep extends React.Component {
                                 : ''}
                         />
                         <ExpensiTextInput
-                            autoCompleteType="new-password"
+                            autoCompleteType="password"
```

Comment:
So this fixes android? What is `new-password` and `password` and what's the difference between them?

Reply:
For TextInput in Android, the autoCompleteType has to be one of the values listed in this link:

https://reactnative.dev/docs/textinput#autocompletetype-android

If you see `password` exists to denote the password field. It isn't an id/name field where we can put `new-password`